earth_enterprise/docker/image-definition/scripts/install-gee.sh

```diff
@@ -7,23 +7,30 @@
 
 # Prepare the install package:
 
-# Download tutorial data:
-# cd earthenterprise/earth_enterprise/tutorial && \
-# mkdir -p FusionTutorial && \
-# wget http://data.opengee.org/FusionTutorial-Full.tar.gz && \
-# tar -xvzf FusionTutorial-Full.tar.gz -C FusionTutorial || exit 1
-
 # Copy binaries to install package location:
-# cd .. && \
-cd earthenterprise/earth_enterprise && \
-scons -j8 release=1 stage_install || exit 1
+# cd earthenterprise/earth_enterprise || exit 1
+# scons -j8 release=1 stage_install || exit 1
+
+
 
-# Install Fusion:
-cd src/installer &&
-(echo C; echo C) | ./install_fusion.sh -hnmf || exit 1
+if type apt-get >/dev/null 2>&1; then
+    # Use RPM packages:
```

Comment:
:stuck_out_tongue: Fix comments to correspond to the type of packages being installed.